tasks/collect-mrrc-params/tests/test-collect-mrrc-params.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-collect-mrrc-params
+spec:
+  description: |
+    Run the collect-mrrc-params task
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > "$(workspaces.data.path)/data.json" << EOF
+              {
+                "mrrc": {
+                  "product": {
+                    "name": "activemq",
+                    "version": "0.0.1"
+                  },
+                  "awsSecret": "charon-aws-credentials",
+                  "environment": "dev",
+                  "release": "ga"
+                },
+                "cosignPubKeyFile": "test-consign.pub",
```

Comment:
Any more thought about putting this and the two below it in the mrrc top level key?

Reply:
Oh, sorry this cosignPubKeyFile is not needed anymore and I'll remove it. But the cosignPubKeyConfig is actually not related to mrrc in business level. It is just used to verify the source maven repo zip, so that's why I put it out of the mrrc.

Reply:
So should we have a maven top level key (like we have mrrc) with this in it? I am just a little wary of putting new top level keys. It is a little confusing to users to have them. I think most if not all of our top level keys have nested keys below them to create a hierarchy style

Reply:
Ok. I'll try to add it.

Reply:
After consideration, I've moved this one under the "mrrc.product" name as it should be bundled to a product. Fixed in new code.